autoextract/__version__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.1.1'
```

Comment:
could you please change it back to 0.1? It is usually better to handle version updates separately from feature pull requests.

Reply:
Sure! Makes sense. Done